### PR TITLE
Update release notes for 2.2.0-rc2

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

**Comment by tbrisker** [comment]
This doesn't seem right, there were several more issues merged since rc1 - but perhaps after you ran the script.

**Comment by upadhyeammit** [comment]
Updated it and added hammer changelog

File: _includes/manuals/2.2/1.2_release_notes.md [hunk 1]
```diff
@@ -32,6 +32,23 @@ Work has been started on creating a new host detail page. The initial implementa
 * Support fetching all records in API ([#29909](https://projects.theforeman.org/issues/29909))
 * graphql: add queries and mutations for settings ([#28920](https://projects.theforeman.org/issues/28920))
 
+#### Hammer CLI
```
Comment (tbrisker) [inline _includes/manuals/2.2/1.2_release_notes.md]: The sections are ordered alphabetically, can you move this section to between `Foreman` and `Installer` sections?

Comment (upadhyeammit) [reply _includes/manuals/2.2/1.2_release_notes.md]: Okay, I thought it should be after API section as per the 2.0 https://github.com/theforeman/theforeman.org/pull/1540 pr .

**Comment by tbrisker** [comment]
There was one issue added to the release after this pr was last update, can you please rerun the script to get it in @upadhyeammit ?

**Review by tbrisker — APPROVED** [review]
Thanks @upadhyeammit !

**Pull request merged**